bin/scripts_wrapper.sh

```diff
@@ -19,6 +19,10 @@ function dr-upload-model {
   dr-update-env && ${DIR}/scripts/upload/upload-model.sh "$@"
 }
 
+function dr-car-upload {
```

Comment:
Could we call it dr-upload-car-zip or dr-create-car-zip? (Or similar...)

Reply:
changed to dr-upload-car-zip